src/Symfony/Component/Translation/Bridge/Loco/Provider/LocoProvider.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Bridge\Loco\Provider;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpFoundation\Response;
```

Comment:
is http-foundation really a dep here?
anyway, we don't use these HTTP consts in the codebase